modules/utils.py

```diff
@@ -185,3 +184,19 @@ def clamp_integer(n):
     if int(n) < 0:
         return None
     return min(BIGQUERY_MAX_INT, int(n))
+
+
+# given a list of integer columns, clamp data to utils.BIGQUERY_MAX_INT and log violations
+def clamp_integers(data, columns):
+    violations = {}
+    for k, v in data.items():
+        if k in columns and v and int(v) > BIGQUERY_MAX_INT:
+            violations[k] = v
+            data[k] = clamp_integer(v)
+    if violations:
+        logging.warning(f"Clamping required for {violations}. data={data}")
+
+
+def columns_for_schema(schema_name):
```

Comment:
Should this be called `int_columns_for_scheme`? Was VERY confused how this was limited to just int columns initially.

Reply:
Yes! Updated.